manifests/startup.pp

```diff
@@ -40,6 +40,14 @@
       fail('you have to provide a pidfile param')
     }
     contain ::systemd
+    case $::osfamily {
```

Comment:
can you please use the new facts hash?
https://docs.puppet.com/puppet/4.10/lang_facts_and_builtin_vars.html